docs/editor/vscode-web.md

```diff
@@ -17,67 +17,77 @@ VS Code for the Web runs entirely in your web browser, so there are certain limi
 
 If you're already in VS Code for the Web at [https://vscode.dev](https://vscode.dev), you can alternatively navigate to different repos via the [Remote Repositories](https://marketplace.visualstudio.com/items?itemName=ms-vscode.remote-repositories) extension commands. Select the remote indicator in the lower left of the Status bar, and you'll be presented with the **Open Remote Repository...** command.
 
 ![GitHub Repositories](images/vscode-web/remote-repositories.png)
 
-GitHub Repositories is the core component that provides the ability to remotely browse and edit a repository from within the editor.
-
-Rather than cloning your work, GitHub Repositories creates a virtual file system to access repositories and pull requests, allowing you to become productive quickly without needing to pull code onto your local machine. You can learn more about the extension in our [GitHub Repositories](/docs/sourcecontrol/github.md#github-repositories-extension) guide.
-
->**Note**: The GitHub Repositories extension works in VS Code Desktop as well to provide fast repository browsing and editing.
-
-You can create a new file in the web just as you would in a VS Code Desktop environment, using **File** > **New File** from the Command Palette (`kbstyle(F1)`).
-
-## Azure Repos (preview)
+## Azure Repos
 
-Support for Azure Repos (part of Azure DevOps) in Visual Studio Code for the Web is in preview, and the experience will continue to grow and evolve over time.
+Just like you can navigate to remote GitHub repos in VS Code for the Web, you can open Azure Repos too.
 
-When you navigate to a URL with the schema `https://vscode.dev/azurerepos/organization/project/repo`, you will be able to read, search the files in the repo, and commit your changes to Azure Repos. You can fetch, pull, and sync changes, and view branches.
+When you navigate to a URL with the schema `https://vscode.dev/azurerepos/<organization>/<project>/<repo>`, you will be able to read, search the files in the repo, and commit your changes to Azure Repos. You can fetch, pull, and sync changes, and view branches.
 
 You can open any repository, branch, or tag from Azure Repos in VS Code for the Web by prefixing `vscode.dev` to the Azure Repos URL.
 
-Alternatively, when you are on an Azure DevOps repository or pull request, you can press (`kbstyle(.)`) to open it in vscode.dev.
+Alternatively, when you are on an Azure DevOps repository or pull request, you can press (`kbstyle(.)`) to open it in VS Code for the Web.
+
+## More custom routes
 
-### Current limitations
+Like in the desktop, you can customize VS Code for the Web through a rich ecosystem of extensions that support just about every back end, language, and service. Unlike in the desktop, it's easy for us to deliver customized experiences with pre-installed extensions through unique `vscode.dev` URLs, aka "routes."
```

Comment:
The wording around "unlike in the desktop..." feels strange to me. Perhaps something like this?

```suggestion
Like in the desktop, you can customize VS Code for the Web through a rich ecosystem of extensions that support just about every back end, language, and service. `vscode.dev` has several built-in routes that provide shortcuts to common experiences.
```

Also, do we need to teach the user the word "route" or can we just use "URLs" or "custom URLs"?

Reply:
I also didn't love the "Unlike desktop" wording, so I love an alternative here!

I like your example. I updated to: 

> `vscode.dev` also allows for custom URLs that provide shortcuts to common experiences.

Let me know what you think.